app/views/catalog_record_ids/_row.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="mb-3 row plain-container">
+<div class="mb-3 row plain-container" <%= 'hidden' if hidden %>>
```

Comment:
Added this to help persist Folio catalog links when in Symphony mode, and vice versa.